### DISCUSSION TITLE
How to disable "select window manager" window?

### DISCUSSION THREAD

**Discussion by steambsd** [lxqt/lxqt#1959] [Q&A] [answered]
Hi! My name is Dmitry. I have autoinstall FreeBSD 13. How to disable first window (select window manager)?

**Comment by stefonarch** [comment, marked as answer]
This window should only appear if no window manager is set yet, doesn't it work like that?

edit: read below.

**Reply by steambsd** [reply]
I agree, but i just want to skip this selection window (for automatic install my scripts). Where the same should be the presets that are before the "startx" command

**Reply by stefonarch** [reply]
I'm afraid you would need to change code of  lxqt-session https://github.com/lxqt/lxqt-session/blob/master/lxqt-session/src/wmselectdialog.cpp

You could modify` /etc/xdg/lxqt/session.conf`  adding already an window manager setting
`window_manager=xyz
`

**Reply by steambsd** [reply]
Okay, tomorrow I'll try it.

YES! It work. Thank you!
